README.md

```diff
@@ -1,24 +1,190 @@
+- Login and Sign Up features implemented using **Devise**
+- Allows users to record and create new Categories and Transactions
+- Breaks down spending for better analysis
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Deployment <a name="loom-demo"></a>
+
+- [Video Presentation](https://drive.google.com/file/d/1kNJ0GnsSbrA2dwnN8_PcfoyBpPj3OAY5/view?usp=sharing)
+
+- [Demo link](https://smartspendr-budget-app.onrender.com/)
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To start and run this project
+```
+bundle install
+```
+```
+rails s
+```
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```
+    ruby >= 3.2.0
+    rails >= 7.0
+    postgres >- 15.3
+```
+As well as:
+
+- IDE to edit and run codes.
+- Git.
+- PostgreSQL installed
+
+
+### Install & Setup
+
+To setup and install this project, follow the below steps:
+- Clone this project by the command:
+
+```
+$ git clone git@github.com:sunga12/Smartspendr.git
+```
+
+- Then switch to the project folder:
+
+```
+$ cd Smartspendr
+```
+
+- Install the required dependencies with `bundle install`
+- Install npm dependencies with `npm install`
+- Run `rails db:setup`
+- Run `rails db:migrate`
+- With Ruby on Rails and PostgreSQL installed in your environment, you can run `rails server`, `rails s` or `./bin/dev` to run the website.
+
+### Tests <a name="run-tests"></a>
+
+- Run `rails db:seed RAILS_ENV=test` to seed the testing database
+- Run `rspec`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Sungabanja Thawethe**
+
+- GitHub: [@Sunga12](https://github.com/sunga12)
+- LinkedIn: [Sungabanja Thawethe](https://linkedin.com/in/sungabanja-thawethe)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Graph based Analysis and Tracking**
+- [ ] **Add more styling to improve the UI**
+- [ ] **Desktop Version**
+- [ ] **Side menu Functionality**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, kindly leave a star!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- We would like to thank Microverse for providing the project specifications as well as the lessons leading up to the project that made completing it possible.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Please make sure to give the appropriate credits to the author of the design in this section of your README! 🤓 🔧 

![image](https://github.com/sunga12/Smartspendr/assets/77069011/cc46c1d6-4728-4919-acd2-30d612aec4bb)